### ISSUE TITLE
Add height/width attributes for customizations

### ISSUE THREAD

**Issue by ppeyro** [predixdesignsystem/px-slider#19]
Currently, other than styling the div, there is no way to customize the slider's height/width.

Future enhancement?

### **Code Snippet:Current styling approach**
```
<div id="thirteenthone" style="width:320px">
      <px-slider...> </px-slider>

```

Screenshot:

![screen shot 2017-04-12 at 2 23 07 pm](https://cloud.githubusercontent.com/assets/10835611/24980105/a246664a-1f8b-11e7-89ad-56ee47afbf2c.png)

### **Code Snippet: Nice to have...**

```
<px-slider
        height=200
        width=320
        value="100"
        ...
      >
```

**Comment by angelonakos** [comment]
I was using the px-slider in my application. The sliderSVG child of the px-slider would have a hardcoded width of 26 when initially loaded and thus my slider would appear wrong. Only if my window was dragged by the cursor (__resized__) would the slider get the desired width/height dimensions. Thus, in order to solve this issue of the slider having a wrong initial width, I would dispatch a resize event ("window.dispatchEvent(new Event('resize'));") and set a timeout when performing this event. 

tl;dr: this issue gave me a bit of pain but I "hacked" it by firing a resize event
